cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -127,6 +122,95 @@ func NewFromCluster(client clientset.Interface, certificatesDir string) (*Client
 	return etcdClient, nil
 }
 
+// getEtcdEndpoints returns the list of etcd endpoints.
+func getEtcdEndpoints(client clientset.Interface) ([]string, error) {
+	return getEtcdEndpointsWithBackoff(client, constants.StaticPodMirroringDefaultRetry)
```

Comment:
Again, can we merge `getEtcdEndpoints` and `getEtcdEndpointsWithBackoff`?

Reply:
The reason for these functions is that `getEtcdEndpoints` and `getAPIEndpoint` don't need testing. We test their `WithBackoff` counterparts, where we can control the backoff on the unit tests, so we have faster unit test execution and controlled with the backoff required depending on the test cases we are stubbing.